omc3/model/model_creators/lhc_model_creator.py

```diff
@@ -28,24 +29,56 @@
     TWISS_DAT,
     TWISS_ELEMENTS_BEST_KNOWLEDGE_DAT,
     TWISS_ELEMENTS_DAT,
-    ACCELERATOR_MODEL_REPOSITORY
+    PATHFETCHER, AFSFETCHER,  # GITFETCHER, LSAFETCHER,
+    ACCELERATOR_MODEL_REPOSITORY,
+    MODIFIER_BRANCH,
+    B2_ERRORS_ROOT,
 )
-from omc3.model.model_creators.abstract_model_creator import ModelCreator
+from omc3.model.model_creators.abstract_model_creator import ModelCreator, check_folder_choices
 from omc3.utils import iotools
 
-LHC_REPOSITORY_NAME = "acc-models-lhc"
-
 LOGGER = logging.getLogger(__name__)
 
 
 def _b2_columns() -> List[str]:
-    cols_outer = [f"{KP}{num}{S}L" for KP in ("K", "P") for num in range(21) for S in ("", "S")]
+    cols_outer = [f"{KP}{num}{S}L" for KP in (
+        "K", "P") for num in range(21) for S in ("", "S")]
     cols_middle = ["DX", "DY", "DS", "DPHI", "DTHETA", "DPSI", "MREX", "MREY", "MREDX", "MREDY",
```

Comment:
Not a new ting to this PR but what is the RFM_FREQ, RFM_HARMON, RFM_LAG used for?